src/config/enterprise.rs

```diff
@@ -601,6 +604,13 @@ pub(crate) fn report_configuration(
             opts,
         } = metadata;
 
+        // Get the machine hostname to avoid reporting the same configuration several time
+        // for a given instance of vector.
+        // If running on kubernetes, we take the pod name instead.
+        let hostname = env::var("VECTOR_SELF_POD_NAME").unwrap_or_else(|_| {
```

Comment:
```suggestion
        let hostname = env::var("VECTOR_SELF_NODE_NAME").unwrap_or_else(|_| {
```

Reply:
🤔 didn't you say pod name?

Reply:
Looks like I said node and linked to node. Node name will be the name of the physical/virtual machine any pod is running on pod name would be the equivalent of `get_hostname`